airflow_kjo/kubernetes_job_launcher.py

```diff
@@ -143,13 +135,13 @@ def apply(self, yaml_obj, extra_configuration={}):
                 return True
             else:
                 logging.error(error.body)
-
         return True
 
-    def watch(self, yaml_obj, running_timeout=None):
-        self._validate_job_yaml(yaml_obj)
-        name, namespace = self._get_name_namespace(yaml_obj)
-        job = self.get(yaml_obj)
+    def watch(self, tail_logs_every=None,
+        tail_logs_line_count=100,
+        tail_logs_only_at_end=False, running_timeout=None):
```

Comment:
I think this is helpful to add the parameters on the method. (For style, these should all be indented though that's less important)

Reply:
will run black before merge :)